packages/wonder-blocks-dropdown/components/single-select.stories.js

```diff
@@ -0,0 +1,72 @@
+// @flow
+import React from "react";
+import {StyleSheet} from "aphrodite";
+
+import {SingleSelect, OptionItem} from "@khanacademy/wonder-blocks-dropdown";
+
+export default {
+    title: "Dropdown / SingleSelect",
+};
+
+const optionItems = new Array(1000)
+    .fill(null)
+    .map((_, i) => (
+        <OptionItem
+            key={i}
+            value={(i + 1).toString()}
+            label={`School ${
+                i + 1
+            } in Wizarding World Some more really long labels?`}
+        />
+    ));
+
+type State = {|
+    selectedValue: string,
+|};
+
+class SingleSelectWithCustomStyles extends React.Component<{||}, State> {
+    state = {
+        selectedValue: "",
+    };
+
+    handleChange = (selected: string) => {
+        this.setState({
+            selectedValue: selected,
+        });
+    };
+
+    render() {
+        return (
+            <SingleSelect
+                onChange={this.handleChange}
+                isFilterable={true}
+                placeholder="Select a school"
+                selectedValue={this.state.selectedValue}
+                dropdownStyle={styles.fullBleed}
+                style={styles.fullBleed}
+            >
+                {optionItems}
+            </SingleSelect>
+        );
+    }
+}
+
+const styles = StyleSheet.create({
+    row: {
+        flexDirection: "row",
+    },
+    fullBleed: {
+        width: "100%",
+    },
+});
+
+export const customStyles = () => <SingleSelectWithCustomStyles />;
```

Comment:
Good call, I'll change that. About hooks, I thought about that, but I'd prefer to keep using classes as examples here in case folks using Webapp can refer to these snippets in a way they can directly use in Webapp.

I'll gladly replace these examples once we fully enable hooks in Webapp.

Re: aside: I'll be reusing those stories in my next PR (Storybook Docs)... wait for it :)